.github/workflows/tests.yml

```diff
@@ -0,0 +1,18 @@
+name: Run tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
IDK much about this. Will this test PRs or just master?

Reply:
the push should test pushes to master (including merge) and the pull request should test pull requests to master